### PR TITLE
Known issues and release scope for environment def

### PR DESCRIPTION
**Pull request by gcomoretto**
I have added a couples of entries following DM-15074.

Can you please review if the phrasing is appropriate and technically the changes are consistent?

### CODE REVIEW

File: releases/note-source/v16_0.rst [hunk 1]
```diff
@@ -29,6 +29,21 @@ Summer 2018 Release (v16_0)
...
+The scope of the release is to be executed on a controlled environment.
+It has been proven to run on newinstall.sh environment, and it is expected
+to run without problems on a package baseline that corresponds to what 
+installed by newinstall.sh version 16.0.
+
+Any deviation from that baseline may imply that that the results are 
+different from the expected ones, i.e. running the corresponding 16.0
+demo package.
+
+
```
Comment (jonathansick) [inline releases/note-source/v16_0.rst]: extra line.

File: releases/note-source/v16_0.rst [hunk 2]
```diff
@@ -29,6 +29,21 @@ Summer 2018 Release (v16_0)
 These release notes highlight significant changes to the Science Pipelines codebase which are likely to be of wide interest.
 For a complete list of changes, see :ref:`release-v16-0-tickets`.
 
+.. _release-v16-0-scope:
+
+Release 16.0 Scope
+------------------
+
+The scope of the release is to be executed on a controlled environment.
```
Comment (jonathansick) [inline releases/note-source/v16_0.rst]: Undo the hard wrapping.

File: releases/note-source/v16_0.rst [hunk 3]
```diff
@@ -29,6 +29,21 @@ Summer 2018 Release (v16_0)
...
+It has been proven to run on newinstall.sh environment, and it is expected
```
Comment (jonathansick) [inline releases/note-source/v16_0.rst]: I'm not entirely sure what's being said here. This is fairly legalistic language that's obfuscating the actual issue and preventing the user from making an appropriate decision.

I think what you're trying to say is "This release is only known to work if you install it according to the <Install with newinstall.sh and eups distrib> instructions." And add the demo package failure as a known issue.

File: install/prereqs/index.rst [hunk 4]
```diff
@@ -50,8 +50,8 @@ The LSST Science Pipelines require Python 3.6 or newer.
 Both the :doc:`newinstall.sh <../newinstall>` and :doc:`lsstsw <../lsstsw>`\ -based installation methods provide dedicated Miniconda environments pre-loaded with Python dependencies.
 If you opt to use your own Python, you can re-create the default Python environment made by :command:`newinstall.sh` and ``lsstsw`` with these Conda environments:
 
-- `macOS <https://github.com/lsst/lsstsw/blob/master/etc/conda3_packages-osx-64.txt>`_.
-- `Linux <https://github.com/lsst/lsstsw/blob/master/etc/conda3_packages-linux-64.txt>`_.
+- `macOS <https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-osx-64.txt>`_.
+- `Linux <https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-linux-64.txt>`_.
```
Comment (jdswinbank) [inline install/prereqs/index.rst]: Being strictly accurate, these package lists are *not* what you get if you use `newinstall.sh`.

Specifically, version 16.0 of `newinstall.sh` [pulls in package lists from lsstsw `10a4fa6`](https://github.com/lsst/lsst/blob/16.0/scripts/newinstall.sh#L42).

However, the SHA1 corresponding to the package lists  tagged `16.0` is actually [`dfec933`](https://github.com/lsst/lsstsw/blob/16.0/etc/conda3_packages-osx-64.txt).

The differences are minor (basically, the addition of the h5py package), but strictly speaking we are not testing against the environment you describe above.

File: known-issues.rst [hunk 5]
```diff
@@ -16,7 +16,10 @@ Binary installation issues
 Cross Platform
 --------------
 
-*No known issues.*
+- The installation is only tested with the Conda environment provided by the <newinstall.sh> 
+  or <lsstsw> installation methods. You may attempt to install ``lsst_distrib`` without this 
+  Conda environment provided that you also install the packages listed in :ref:`prereq-platforms`, 
+  though these installations may not behave as expected. See :jira:`DM-15040` and :jira:`DM-15041`.
```
Comment (jdswinbank) [inline known-issues.rst]: I don't think @jonathansick will approve of the hard line wrapping (& I hope he'll forgive me if I'm wrong). Also, it looks like there is some trailing whitespace on these lines.

Other than that, I don't love the above wording, but I think it's ok given the apparent urgency of this release. I hope we can do better next time.

Comment (jonathansick) [reply known-issues.rst]: How about this?

```
- This release may not work when installed without the Conda environments provided by the documented :doc:`newinstall.sh </install/newinstall>`, :doc:`lsstsw </install/lsstsw>`, or :doc:`Docker <docker>` installation methods.
  The :ref:`documented Python dependencies <python-deps>` may differ slightly from those in the release, causing issues with the :doc:`validation demo </install/demo>`.
  See :jira:`DM-15040` and :jira:`DM-15041` for more discussion.
```

Does this capture the issue appropriately?

File: install/prereqs/index.rst (hunk 4 above)
Comment (jhoblitt) [reply install/prereqs/index.rst]: Please us `10a4fa6` as the git ref as this was/is the hard-coded value used for the tarball build.

**Pull request merged**